tests/FakeItEasy.Specs/ThrowsAsyncSpecs.cs

```diff
@@ -48,7 +48,7 @@ public static void NoResultPassExceptionInstance(IFoo fake, Task task)
                 .x(() => task.Status.Should().Be(TaskStatus.Faulted));
 
             "And the task's exception is the configured exception"
-                .x(() => task.Exception?.InnerException.Should().BeAnExceptionOfType<MyException>());

```

Comment:
Ah, sorry. I wanted to comment on it, but I forgot.

This code was wrong, because the `?.` short-circuits the whole expression, so `.Should().BeAnException()` would not be executed if `task.Exception` was null.